master/reference/calicoctl/resources/networkpolicy.md

```diff
@@ -7,7 +7,12 @@ A network policy resource (`NetworkPolicy`) represents an ordered set of rules w
 to a collection of endpoints that match a [label selector](#selector).
 
 NetworkPolicy is a namespaced resource. NetworkPolicy in a specific namespace
-only applies to [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint) in that namespace.
+only applies to [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint)
+in that namespace. Two resources are in the same namespace if the namespace
+value is set the same on both.
+Select a namespace in a NetworkPolicy in the standard selector by using
```

Comment:
I I think this line is wrong - given these are namespaced, adding a `projectcalico.org/namespace` label selector won't select anything, because you can only select things _in the same namespace_.

Reply:
It should probably go in the GNP doc

Reply:
I think @robbrockbank said that if the selector contains the namespace label string, the namespace  selector will not be added. or @gunjan5 you were part of that conversation too, does that sound right?

Reply:
If the selector contains the string `pncs.` then it won't be added, but that's just a hack for the alpha and we NEED something better. 

I think better just to not document that for now.

the `projectcalico.org/namespace` label is something else